<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>798</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3000</width>
    <height>3000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Видео АРМ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="modulesFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>321</width>
      <height>451</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>900</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="moduleWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>431</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>900</height>
      </size>
     </property>
     <widget class="QFrame" name="modulesFrame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>321</width>
        <height>441</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>1000</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="settingsBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>221</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Настройки</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Загрузки/configuration 1.png</normaloff>../../Загрузки/configuration 1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="filesBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>298</y>
         <width>221</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Файловая система</string>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="reportBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>238</y>
         <width>221</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Донесения</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Загрузки/report_writing_0515-0909-2722-3528_SMU.png</normaloff>../../Загрузки/report_writing_0515-0909-2722-3528_SMU.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="videoBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>178</y>
         <width>221</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Видеопроигрыватель</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Загрузки/video-player-icon.png</normaloff>../../Загрузки/video-player-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="camBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>118</y>
         <width>221</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Видео и управление ЦН</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Загрузки/camera.png</normaloff>../../Загрузки/camera.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="statusMsg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>746</y>
      <width>1366</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Приложение запущено</string>
    </property>
   </widget>
   <widget class="QFrame" name="smallFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>600</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>0</y>
      <width>731</width>
      <height>751</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="tool1">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>0</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>200</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="tool2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>200</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="tool3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>400</y>
      <width>300</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>370</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="changeBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Загрузки/exchange.png</normaloff>../../Загрузки/exchange.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1362</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>помощь</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
